pluma/stream/zeromq.py

```diff
@@ -206,4 +226,30 @@ def __init__(self, eventcode: int, **kw):
                  ('Subject.Position.Y', np.single),
                  ('Point.Position.X', np.single),
                  ('Point.Position.Y', np.single)]],
+            **kw)
+class ProtocolNewSceneStream(ZmqStream):
+    def __init__(self, eventcode: int, **kw):
+        super(UnityNewSceneStream, self).__init__(
```

Comment:
```suggestion
        super(ProtocolNewSceneStream, self).__init__(
```

The type name in the call to the parent constructor using `super` have to be renamed as well otherwise the code will throw. This applies to every renamed class.

Reply:
Solved in 0d6c0e9